src/Service/Migrate/DownService.php

```diff
@@ -37,29 +42,37 @@ public function __construct(
      */
     public function run(string $class): bool
     {
-        $this->consoleHelper->io()->title("\nReverting $class");
+        if ($this->io) {
```

Comment:
Is it possible to avoid this and alike `if`?

Reply:
Potentialy user of package can use `DownService` not in console. Output to console - optional.